README.md

```diff
@@ -1023,6 +1024,53 @@ export default class MyComponent extends Vue {
 </script>
 ```
 
+### 8.6 Vuex
+
+Basically, vuex is like a "middleware" it is responsible for being the intermediary between the API and the Components.
+
+`API => Vuex => Components`
+
+![Modelo](./assets/image/modelos.png)
+
+#### VUEX attributes
+
+- state:{} - controls the state;
+- mutations:{} - change the state;
+- actions:{} - actions within a given context;
+- modules:{} - single divisions that contain `states, mutations and actions` watch out for certain modules
```

Comment:
Suggestion: 

- state:{} - reactive state;
- Getters: {} - responsible for returning specific fields from the state or/and handle them before being readed.
- mutations:{} - responsible for change the state;
- actions:{} - responsible for handle logic and call mutations to change the state;